source/moon-resolution.js

```diff
@@ -69,6 +71,10 @@
 			type = type || screenType;
 			if (type) {
 				enyo.dom.addBodyClass('moon-res-' + type.toLowerCase());
+				var scrObj = getScreenTypeObject(type);
+				if (scrObj.aspectRatioName) {
+					enyo.dom.addBodyClass('enyo-aspect-ratio-' + scrObj.aspectRatioName.toLowerCase());
```

Comment:
This is intentionally different than the `moon-` namespace above it, to more-future-proof the developers' work that uses this class.